vm/hugepage/libhugetlbfs/README.md

```diff
@@ -0,0 +1,13 @@
+# libhugetlbfs test suite
+libhugetlbfs test provides huge pages of memory unit testing for the libhugetlbfs file system. The source code for libhugetlbfs wrapper can be found at https://github.com/libhugetlbfs/libhugetlbfs.git \
```

Comment:
@ftjahjad can you please remove the `\` at the end of the line here? I don't see any use for it

Reply:
@Veruu 
`\`  at the end  to align "test maintainer......." to next line on my understanding,
If  without `\`, "test maintainer...." will be display next after it, not on next line :-D

Reply:
Ah, ok! I missed there is no empty line so I thought it's joining the lines together. My bad.

Reply:
Thanks  @revuu :+1: